comit/src/network/protocols/orderbook/order.rs

```diff
@@ -35,27 +34,38 @@ impl FromStr for OrderId {
 pub struct Order {
     pub id: OrderId,
     pub maker: MakerId,
+    pub trade: Trade,
     #[serde(with = "asset::bitcoin::sats_as_string")]
-    pub buy: asset::Bitcoin,
+    pub bitcoin_amount: asset::Bitcoin,
     pub bitcoin_ledger: ledger::Bitcoin,
-    pub sell: asset::Erc20,
+    pub ethereum_amount: asset::Erc20Quantity,
+    pub token_contract: identity::Ethereum,
     pub ethereum_ledger: ledger::Ethereum,
+    // TODO: Add both expiries
     pub absolute_expiry: u32,
 }
 
 impl Order {
-    pub fn topic(&self) -> Topic {
-        let peer_id = PeerId::from(self.maker.clone());
-        TradingPair {
-            buy: SwapType::Hbit,
-            sell: SwapType::Herc20,
-        }
-        .to_topic(&peer_id)
+    pub fn tp(&self) -> TradingPair {
+        TradingPair::BtcDai
     }
+
+    pub fn topic(&self) -> String {
+        BTC_DAI.to_string()
+    }
+}
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum Trade {
```

Comment:
Nectar calls this a `Position` IIRC.

What is the motivation for diverging here?

Reply:
I was not aware of that term. Will use it.